NetCDF/exportNetCDF.m

```diff
@@ -470,7 +512,7 @@
 %
   switch(type)
     case 'dimensions'
-      var = sample_data.dimensions{varIdx};
+      var = sample_data.dimensions{varIdx}; % when is this even triggered? (Loz, 20230624)
```

Comment:
@ggalibert I wouldn't mind your input on this? I tried debugging and couldn't see this line being ever reached

Reply:
Probably never, if you look for where addQCvar function is called, you will see it is not called for dimensions.